pkg/ui/assets/caret-left.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+
+// tslint:disable-next-line: variable-name
+const CuretLeft = () => (
+  <svg width={11} height={16} viewBox="0 0 11 16" fill="none">
```

Comment:
Why do we need to put the svg into a React component? Is there no way to import and use as-is? Does babel have an svg-loader or something? Maybe this? https://webpack.js.org/loaders/svg-inline-loader/